src/test/java/seedu/taskify/logic/parser/EditCommandParserTest.java

```diff
@@ -105,18 +98,18 @@ public void parse_invalidValue_failure() {
         assertParseFailure(parser, "1" + TAG_EMPTY + TAG_DESC_FRIEND + TAG_DESC_HUSBAND, Tag.MESSAGE_CONSTRAINTS);
 
         // multiple invalid values, but only the first invalid value is captured
-        assertParseFailure(parser, "1" + INVALID_NAME_DESC + INVALID_ADDRESS_DESC
+        assertParseFailure(parser, "1" + INVALID_NAME_DESC
                 + VALID_DESCRIPTION_AMY, Name.MESSAGE_CONSTRAINTS);
     }
```

Comment:
similar to a previous comment, should change the description to an invalid one

Reply:
done